README.md

```diff
@@ -1,5 +1,7 @@
 # HAPI-FHIR Starter Project
 
+A fork of [hapi-fhir-jpaserver-starter](https://github.com/hl7-eu/hapi-fhir-jpaserver-starter) with added support for research study cohorting operations.
```

Comment:
Again, no need to specify this is a fork, just add the possible features to the appropriate section